aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -486,7 +488,62 @@ Tensor dense_to_sparse_bsc(const Tensor& self, IntArrayRef blocksize) {
   return self;
 }
 
+static inline Tensor sparse_csr_transpose(const Tensor & self) {
```

Comment:
This block was moved from TensorShape.cpp and will be removed when changing transpose to return CSC instead of CSR when given CSR.